modules/arrow/test/arrow-loader.spec.js

```diff
@@ -56,7 +56,9 @@ test('ArrowLoader#parse (WORKER)', async t => {
     return;
   }
 
-  const data = await parse(fetchFile(ARROW_SIMPLE), ArrowWorkerLoader);
+  const data = await parse(fetchFile(ARROW_SIMPLE), ArrowWorkerLoader, {
+    workerUrl: 'modules/arrow/dist/arrow-loader.worker.js'
```

Comment:
Nit: Add comment explaining why we override workerUrl. (I assume it is because we don't want to be CDN dependent in unit tests).

Reply:
If we fix a bug in the loader we need to test the local version, not the one already published to cdn.